src/test/K6/src/tests/platform/events/subscriptions.js

```diff
@@ -57,6 +57,8 @@ export default function (data) {
   });
   addErrorCount(success);
 
+  sleep(20);
```

Comment:
would it not make sense to have the delay before line 54 instead?

Reply:
the sleep is added just before the api call to delete the subscription. Line 54 is the test on response from get subscription.